filters/quantitative_ques/filter.py

```diff
@@ -11,7 +11,16 @@ def __init__(self):
         super().__init__()
         self.nlp = spacy.load("en_core_web_sm")
         # Covers the broad types of quant questions: distance , age , measurable , un-measurable
-        self.quant_ques = ['many','far','close','long','much','old']
+        self.quant_ques = ['many','much',
+                           'close','far',
+                           'young','old',
+                           'short','tall'
+                           'heavy','light'
+                           'narrow','wide',
+                           'deep','shallow',
+                           'broad','thin',
+                           'near','long']
+
```

Comment:
Thanks for expanding! And nvm the expansion didn't change things here, maybe it will matter more on actual quant-rich benchmarks [i guess frac is low in squad anyway]